src/Libraries/Microsoft.Extensions.Telemetry.Abstractions/buildTransitive/net6.0/Microsoft.Extensions.Telemetry.Abstractions.targets

```diff
@@ -13,4 +13,19 @@
       <Analyzer Remove="@(_Microsoft_Extensions_Logging_AbstractionsAnalyzer)" />
     </ItemGroup>
   </Target>
-</Project>
+  <Target Name="_Microsoft_Extensions_Logging_AbstractionsRemoveAnalyzersWPF"
```

Comment:
Do we understand why this only repros for WPF projects? It sounds like maybe we may not be sure yet. I wonder if it has to do with the fact that they use a different TFM (as WPF uses netx.y-windows), and so build targets are not imported correctly. Have we validated that this fix works in a WPF app?

Reply:
> Do we understand why this only repros for WPF projects?

I _think_ that's because WPF generates a temporary project during the build phase to compile xaml, and that temp project isn't importing/executing the targets we expect. @xakep139 please correct me, if I got it wrong.

Reply:
That's my understanding as well. There's an additional `RemoveDuplicateAnalyzers` target that is added when temp project is compiled